WPMedia/fusion-news-theme-blocks
# TBRANDS-50: Pull content-api into v2.00

## Description

Update some minor things to bring the content-api into v2.

**This currently fails the test because there are no tests for the transform which will be changing here with resizer v2.  I did not see the purpose of testing this until that update occurs.**

## Jira Ticket

- [TBRANDS-50](https://arcpublishing.atlassian.net/browse/TBRANDS-50)

## Acceptance Criteria

Due to content blocks needing this API, we need to bring forward the content-api content source and make sure it fits in the v2 architecture if needed.

## Test Steps

1. Checkout this branch `git checkout tbrands-50`
2. Run fusion repo with linked blocks `npx fusion start -f -l`
3. See content-api in Pagebuilder local
![image](https://user-images.githubusercontent.com/2287238/167698802-07d6056e-6fb3-40ef-950b-66b05f4fb8e4.png)

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
